modules/controller/dc-controller.js

```diff
@@ -365,6 +366,51 @@ class DCController {
             dv_node_id,
         );
     }
+
+    /**
+     * Query local data
+     * @param query Query
+     * @returns {Promise<*>}
+     */
+    async queryLocal(query) {
+        this.logger.info(`Local query handling triggered with ${JSON.stringify(query)}.`);
+        const validationError = ObjectValidator.validateSearchQueryObject(query);
+        if (validationError) {
+            throw validationError;
+        }
+
+        const pathsArray = Utilities.arrayze(query[0].path);
+        const valuesArray = Utilities.arrayze(query[0].value);
+        const { opcode } = query[0];
+
+        let response = [];
+        for (let i = 0; i < valuesArray.length; i += 1) {
+            let result =
+                // eslint-disable-next-line no-await-in-loop
+                await this.graphStorage.findLocalQuery({
```

Comment:
Is it possible to pass multiple identifiers, so that we have only one database query. This way we will have multiple queries for each identifier.

Reply:
Good point, agreed